types/debug/index.d.ts

```diff
@@ -1,40 +1,45 @@
-// Type definitions for debug
+// Type definitions for debug 4.1
 // Project: https://github.com/visionmedia/debug
 // Definitions by: Seon-Wook Park <https://github.com/swook>
 //                 Gal Talmor <https://github.com/galtalmor>
 //                 John McLaughlin <https://github.com/zamb3zi>
 //                 Brasten Sager <https://github.com/brasten>
+//                 Nicolas Penin <https://github.com/npenin>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare var debug: debug.IDebug;
+declare var debug: debug.Debug & {debug: debug.Debug, default: debug.Debug};
```

Comment:
This is an interesting change. The `default` property here indicates either 1) I can call and use `debug.default` or 2) I can use the module as an ES6 module with a default import. I don't see any docs that have this usage - maybe you can add some tests that exercise this usage pattern?